Commands/Taxonomy/ImportTermSet.cs

```diff
@@ -19,9 +19,9 @@ namespace SharePointPnP.PowerShell.Commands.Taxonomy
 
 It is recommended that a fixed GUID be used as the termSetId, to allow the term set to be easily updated (so do not pass Guid.Empty).
 
-In contrast to the web interface import, this is not a one-off import but runs synchronisation logic allowing updating of an existing Term Set. When synchronising, any existing terms are matched (with Term Description and Available for Tagging updated as necessary), any new terms are added in the correct place in the hierarchy, and (if synchroniseDeletions is set) any terms not in the imported file are removed.
```

Comment:
Like my comment earlier, this is correct English as such, and synchronization is only used in American English